client/src/components/creative-collab.css

```diff
@@ -1,12 +1,30 @@
-.editor,
+.editor {
+  width: 95%;
+  height: 200px;
```

Comment:
Try to stick with one unit (either px or percentage). Probably, you would go with % in order to keep it responsive.